lib/livebook/runtime/erl_dist/node_manager.ex

```diff
@@ -78,6 +78,7 @@ defmodule Livebook.Runtime.ErlDist.NodeManager do
   def init(opts) do
     unload_modules_on_termination = Keyword.get(opts, :unload_modules_on_termination, true)
     auto_termination = Keyword.get(opts, :auto_termination, true)
+    parent_node = Keyword.get(opts, :parent_node)
```

Comment:
Could you please document this option on the `start/1` function above and that we disconnect from the node on termination? :)